sdk/azurestack/Microsoft.AzureStack.Management.Fabric.Admin/src/Properties/AssemblyInfo.cs

```diff
@@ -15,5 +15,5 @@
 [assembly: AssemblyCulture("")]
```

Comment:
we typically update the test csproj and assemblyinfo as well for version changes

Reply:
Updated to 2.9.0-preview and 2.9.0.0